src/views/Lottery/components/PreviousRoundCard/FooterExpanded.tsx

```diff
@@ -20,20 +21,43 @@ const NextDrawWrapper = styled(Flex)`
   }
 `
 
-const PreviousRoundCardFooter: React.FC<{ lotteryData: LotteryRound; lotteryId: string }> = ({
-  lotteryData,
+const PreviousRoundCardFooter: React.FC<{ lotteryNodeData: LotteryRound; lotteryId: string }> = ({
+  lotteryNodeData,
   lotteryId,
 }) => {
   const { t } = useTranslation()
+  const [fetchedLotteryGraphData, setFetchedLotteryGraphData] = useState<LotteryRoundGraphEntity>()
   const lotteryGraphData = useGetLotteryGraphDataById(lotteryId)
   const cakePriceBusd = usePriceCakeBusd()
 
+  useEffect(() => {
+    const getGraphData = async () => {
+      const fetchedGraphData = await getGraphLotteries(undefined, undefined, { id_in: [lotteryId] })
+      setFetchedLotteryGraphData(fetchedGraphData[0])
+    }
+    if (!lotteryGraphData) {
```

Comment:
If no graph data is returned for the selected round - fetch it. Will be triggered when user is viewing the expanded footer for rounds > 100 ago